migrations/2023-11-29-091848_drop_column_seller_name_payment_link/up.sql

```diff
@@ -0,0 +1,2 @@
+-- Your SQL goes here
+ALTER TABLE payment_link DROP COLUMN seller_name;
```

Comment:
Why did we remove and rename this column?

Reply:
error from my part, fixed it